src/main/resources/application-azure.properties

```diff
@@ -0,0 +1,3 @@
+# Import KeyVault secrets as properties
+spring.cloud.azure.keyvault.secret.property-sources[0].enabled=true
+spring.cloud.azure.keyvault.secret.property-sources[0].endpoint=${AZURE_KEY_VAULT_ENDPOINT}
```

Comment:
Do you need to do this? This was only used by the java ToDO because we do have private configuration stored in Azure KeyVault.. I don't see any code changes in the Java application that has changed to reference new secrets

Reply:
the `MYSQL_PASS` is stored in Azure KeyVault.
https://github.com/wangmingliang-ms/spring-petclinic-java-mysql/blob/7fcd9d8d10285b6d388721fa5c94087a330a4279/src/main/resources/application-mysql.properties#L5

https://github.com/wangmingliang-ms/spring-petclinic-java-mysql/blob/7fcd9d8d10285b6d388721fa5c94087a330a4279/infra/database/mysql.bicep#L97-L103

https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html#secret-management

Reply:
Make sense. I don't have permissions to resolve this comment, so will leave it up to you.